src/components/ContentWrapper.jsx

```diff
@@ -340,12 +341,13 @@ const ContentWrapper = (props) => {
         return <PipelineRedirectToDataProcessing experimentId={routeExperimentId} pipelineStatus='running' />;
       }
 
-      if (process.env.NODE_ENV === 'development') {
-        return children;
+      if (qcStatusKey === pipelineStatusValues.NOT_CREATED
+        && currentModule !== modules.DATA_PROCESSING && !isSeurat) {
+        return <PipelineRedirectToDataProcessing experimentId={routeExperimentId} pipelineStatus='toBeRun' />;
```

Comment:
why is this added and the below one deleted? They are similar but different

Reply:
This `PipelineRedirectToDataProcessing` is supposed to be returned when qc is not created, but it was being returned when gem2s was not created, so I added the check on `qcStatusKey === pipelineStatusValues.NOT_CREATED`.

I changed `pipelineNotCreated` for `gem2sNotCreated` because I think the ambiguous naming of `pipeline` to refer to gem2s/seurat only and not qc makes us prone to problems like this.